RecoHI/HiCentralityAlgos/python/HiClusterCompatibility_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+hiClusterCompatibility = cms.EDProducer("ClusterCompatibilityProducer",
+   inputTag      = cms.InputTag( "siPixelRecHits" ),
+   minZ          = cms.double(-40.0),
+   maxZ          = cms.double(40.05),
+   zStep         = cms.double(0.2)
```

Comment:
this looks really excessive.
Was this step optimized?
The z is used only to compute the incidence angle.
I'd naively think that a step in angle smalle than ~0.2 isn't really necessary; this makes zStep ~ 4cm * 0.2 ~ 0.8 cm good enough.